pulp_rpm/tests/functional/constants.py

```diff
@@ -238,6 +238,46 @@
 Note: This repository uses unsigned RPMs.
 """
 
+RPM_ADVISORY_UPDATED_VERSION_URL = urljoin(
+    PULP_FIXTURES_BASE_URL, 'rpm-updated-updateversion'
+)
+"""The URL to a repository containing Advisories with same ID as the ones in the standard
+unsigned rpm repository, but with updated Advisory version.
+"""
+
+RPM_ADVISORY_DIFFERENT_PKGLIST_URL = urljoin(
+    PULP_FIXTURES_BASE_URL, 'rpm-advisory-diffpkgs'
+)
+"""The URL to a repository containing Advisories with same ID and version as the ones in the
+standard unsigned rpm repository, but with different pkglist.
+"""
+
+RPM_ADVISORY_DIFFERENT_REPO_URL = urljoin(
+    PULP_FIXTURES_BASE_URL, 'rpm-advisory-diff-repo'
+)
+"""The URL to a repository containing Advisories with same ID and version as the ones in the
+standard unsigned rpm repository, but with different update_date and packages intersection.
+"""
+
+RPM_ADVISORY_INCOMPLETE_PKG_LIST_URL = urljoin(
+    PULP_FIXTURES_BASE_URL, 'rpm-advisory-incomplete-package-list'
+)
+"""The URL to a repository containing Advisories with same update_date and version as the ones
+in the standard unsigned rpm repository, but pkglist intersection is non-empty and not equal
+to either pkglist.
+"""
+
+RPM_ADVISORY_NO_DATES = urljoin(
+    PULP_FIXTURES_BASE_URL, 'rpm-advisory-no-dates'
+)
+"""The URL to a repository containing Advisories with same id and version as the ones
+in the standard unsigned rpm repository, but no update_date or issue_date.
+"""
+
+RPM_ADVISORY_TEST_ID = 'RHEA-2012:0056'
+RPM_ADVISORY_TEST_REMOVE_COUNT = 3
+RPM_ADVISORY_TEST_ADDED_COUNT = 6
+
 RPM_UPDATERECORD_ID = 'RHEA-2012:0058'
 """The ID of an UpdateRecord (advisory/erratum).
```

Comment:
Please unify the naming.
If we list here 2 different advisories, the name should highlight what's special about each of them.